yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/tags/IgnoreTag.java

```diff
@@ -36,7 +37,7 @@ public final class IgnoreTag implements CustomTag {
     private static final Tag tag = new Tag("!ignore");
 
     @Nonnull
-    private static final Matchable INSTANCE = (other, rowNumber, cellRef) -> Matchers.ResultSetMatchResult.success();
+    private static final Matchable INSTANCE = new IgnoreMatcher();
```

Comment:
I believe this was added because we believe there's a gap here in the coverage. Should we have a yaml file somewhere to test this out? Something that would have failed before but succeeds now?